schema/url/standalone-map/plugin-collection-markers.raml

```diff
@@ -0,0 +1,81 @@
+#%RAML 1.0 Library
+
+types:
+
+  StandaloneCollectionMapMarkerPostRequest:
+    type: DataPluginRequestBase
+    properties:
+      config: StandaloneCollectionMapMarkerSpec
+
+  StandaloneCollectionMapMarkerSpec:
+    additionalProperties: false
+    description: Specification for a map marker
+    properties:
+      outputEntityId:
+        required: true
+        type: string
+        description: Entity ID associated with collection variable of interest.
+      collectionVariableId:
+        required: true
+        type: VariableSpec
+        description: Collection variable to use for markers.
+      memberVariables:
+        type: MemberVariable[]
+        description: | 
+          Optionally, a set of member variable IDs can be provided. Aggregate output values will be provided per marker
+          for each variable in this set. By default, all variables in collection will be provided. The maximum number of
+          variables we can provide output for per marker is 8.
+
+  # Note that this is not using inheritance as it breaks API doc generation. Instead, both continuous/categorical
+  # configurations are top-level fields and the backend will validate that the config is compatible with the associated var.
+  MemberVariable:
+    additionalProperties: false
+    properties:
+      variableId: string
+      categoricalConfig:
```

Comment:
all members should share the same configuration here, so im not sure it needs to belong to MemberVariable. where you have MemberVariable[] above i think could just be VariableSpec[], then add this config to StandaloneCollectionMapMarkerSpec

Reply:
Oh I see, is this even the case for categorical variables? I'm not that familiar with collections, do all member variables of categorical collections share variable values?

Reply:
Yes it's true of categorical collections as well. It's perhaps the chief defining characteristic of collections that their members share theoretical values.